.gitlab-ci.yml

```diff
@@ -7,8 +7,10 @@ stages:
   - promote-to-prd
   - prd
 
-setup:
+aws-setup:
   stage: setup
+  tags:
+  - aws
```

Comment:
it's only a cosmetic change, but is there a particular reason whay the `tags` identation is different of the `script` section?

Reply:
¯\(°_o)/¯

(This is fixed in my new PR, #92.)